lib/turn.rb

```diff
@@ -0,0 +1,22 @@
+class Turn ()
+  attr_reader :guess, :card
+
+  def initialize(guess, card)
+    @guess = guess
+    @card = card
+  end
+
+#refactor to not need a param
+  def correct?(guess)
```

Comment:
you do not need to pass this method an argument. You can see, even in the method, you don't reference `guess` but you reference `@guess`.. so, you can take off the placeholder here.